Jenkinsfile

```diff
@@ -299,6 +300,19 @@ def job = {
                                 cmd += " -DnexusUrl=s3://staging-ksqldb-maven/maven"
                                 sh cmd
                             }
+                            sh """
+                                set +x
+                                TMP_GPG_PASS=\$(mktemp -t XXXgpgpass)
+                                echo "\${GPG_PASSPHRASE}" > "\${TMP_GPG_PASS}"
+                                bash confluent-process-packages.sh \
+                                    --bucket staging-ksqldb-packages \
+                                    --region us-west-2 \
+                                    --prefix '\$PACKAGE_TYPE/${config.ksql_db_packages_prefix}' \
+                                    --input-dir "\${PWD}/output" \
+                                    --sign-key-file "${gpg_packaging_key}" \
+                                    --passphrase-file "\${TMP_GPG_PASS}"
+                                rm -f "\${TMP_GPG_PASS}"
+                            """
```

Comment:
I'm aware of how "ugly" this part looks, but its due to a requirement of needing to access Jenkins variables inside the script, as well as referencing (escaped) shell environment variables within the script.